apps/explorer/src/components/NetworkSelector/NetworkSelector.styled.ts

```diff
@@ -14,6 +14,7 @@ export const SelectorContainer = styled.div`
   }
 
   ${ArrowIconCSS}
+  color: ${() => Color.explorer_orange1};
```

Comment:
Nitpick: `color: ${Color.explorer_orange1};`

Reply:
Addressed (and every other instance I found)